src/html-bundler.ts

```diff
@@ -57,6 +58,11 @@ export class HtmlBundler {
 
     if (this.bundler.enableScriptInlining) {
       await this._inlineScripts(ast);
+      this.document = await this.bundler.analyzeContents(
+          this.assignedBundle.url, serialize(ast));
+      ast = this.document.parsedDocument.ast;
+      dom5.removeFakeRootElements(ast);
+      await this._rollupInlineScripts();
```

Comment:
maybe `_inlineModuleScripts`, and `_inlineNonModuleScripts` for the old function?

Reply:
Refactored and that made this cleaner.  Check out commit 687a9fcc77ac472563da777a5581a6862eac1ce2 if you're looking for an exciting refactoring.